File: src/javas/proglem/loop_Programs/table_of_any_number_P_08.java
package javas.proglem.loop_Programs;

import java.util.Scanner;

public class table_of_any_number_P_08 {
    public static void main(String[] args) {

        int num,result;
        Scanner input = new Scanner(System.in);

        System.out.print("enter number of table:");
        num = input.nextInt();

        for (int i=1;i<=10;i++){
            result=num * i;
            System.out.println(num+"X"+i+"="+result);
        }


    }
}

File: src/Kotlin/problems/loop Programs/table of any number P_08.kt
package Kotlin.problems.`loop Programs`

import java.util.*

fun main() {
var input = Scanner(System.`in`)
var num=0
var result =0

println("enter number of table:")
num = input.nextInt()
for (i in 1..10){
    result = num * i
    println("$num X $i = $result")
}
}